play_by_play_testing.py

```diff
@@ -0,0 +1,95 @@
+import unittest
+from pandas.testing import assert_frame_equal
+import pandas as pd
+from play_by_play import PlayByPlay
+from config import dtypes
+
+
+class TestPlays(unittest.TestCase):
+
+    def setUp(self) -> None:
+        directory = 'test_data'
+        self.row_1 = pd.read_csv(directory + '/row_1.csv',
```

Comment:
This is good to use CSV data for your test cases.

I'd recommend naming the test cases if you can. I presume each one covers a different test case. If so, describe them to the best of your ability.

Reply:
Name the CSV's that are currently row1.csv, row2.csv... after the test case each is covering?